crm_request/models/request.py

```diff
@@ -72,15 +74,19 @@ def action_reply(self):
             'lang': self.language,
         }
 
-        if self.partner_id:
-            partner = self._get_partner_alias(
-                self.partner_id, parseaddr(self.email_from)[1]
+        if original_partner:
+            alias_partner = self._get_partner_alias(
+                original_partner, parseaddr(self.email_from)[1]
             )
+            partner = alias_partner
+            if alias_partner == original_partner:
+                partner = original_partner
```

Comment:
These two lines don't do anything and are not necessary.

Reply:
I don't think they are not necessary as they permit to simplify a bit the code after, when filtering messages (line 87-89)